js/map.js

```diff
@@ -3,9 +3,15 @@ var infowindow = null;
 var markers = [];
 var currentNodeListURL;
 var circle = null;
+var pingData;
 
 function initialize() {
-	
+
+	//Pull Ping Data
+	$.getJSON("https://node2.e-mesh.net/ping.json.php", function (data) {
```

Comment:
Can we run this local?

Reply:
Local will also need access to hyperborea. 

Other option is to have a remote cjdns node upload the data to the server very run.

Reply:
Also another php script that runs on a hyperborea node that is "stage 2" of this is the linking script.

https://github.com/tomeshnet/tomesh.net/tree/MapMonitoring-Links

https://github.com/darkdrgn2k/ToMeshStuff/blob/master/cjdnsDump.php